.drone.env

```
@@ -3,5 +3,5 @@ CORE_COMMITID=b33853030d88a6b530ab411b86539ac6eb9970b8
 CORE_BRANCH=master
 
 # The test runner source for UI tests
-WEB_COMMITID=262398e513253d3296792434ecc4a96fa1c19a73
-WEB_BRANCH=testSharesFolderAlwaysVisible
+WEB_COMMITID=e84960674723d8fb8c3c42e1ada8ef8cc1747ba6
```

```suggestion
WEB_COMMITID=fbfef7c48676cdc4a492b56f9e07856534910e6a
```

`e84960674723d8fb8c3c42e1ada8ef8cc1747ba6` looks a bit far back in `owncloud/web` - maybe specify the "Automated changelog update" commit id that happened in web master directly after the Release 5.0.0 PR was merged.

---

Done, thanks for the hint!

---

Argh, just seeing now that the referenced branch has test adjustments that are not on web master. Thus we'll probably need to rebase that branch in web to be in sync with current web master again and then reference **that** commit id here instead of one from the current master.
Or - much simpler would be to merge that commit in web into master?! https://github.com/owncloud/web/commit/262398e513253d3296792434ecc4a96fa1c19a73 @phil-davis why is that commit not in master?

---

That branch and commit was at an old point on web so that we could choose to run test code at exactly that point pending the next web release. Those changes to test scenarios are also already in web master - so it should "just work".

---

Ah, nice! So that branch can be deleted now in web?

---

(of course **after** merging this PR)

---

> Ah, nice! So that branch can be deleted now in web?

yes, after we are happy that there is no CI that needs. We can wait an extra day ;)